### PR TITLE
Aether python library

### PR DESCRIPTION
**Pull request by lordmallam**
Using `aether.python` https://github.com/eHealthAfrica/aether-python-library

### CODE REVIEW

File: aether-kernel/aether/kernel/api/validators.py [hunk 1]
```diff
@@ -1,214 +0,0 @@
-# Copyright (C) 2019 by eHealth Africa : http://www.eHealthAfrica.org
```
Comment (shawnsarwar) [inline aether-kernel/aether/kernel/api/validators.py]: This set of functions seems pretty tightly coupled to Django / kernel. Is there much overlap with what's required by the extractor module? Can the Django pieces stay here or are they also used by the extractor module?

File: aether-kernel/aether/kernel/api/views.py [hunk 2]
```diff
@@ -32,14 +32,13 @@
 from rest_framework.renderers import JSONRenderer
 
 from aether.sdk.multitenancy.views import MtViewSetMixin
-
-from .avro_tools import extract_jsonpaths_and_docs
-from .constants import LINKED_DATA_MAX_DEPTH
-from .entity_extractor import (
+from aether.python.entity.extractor import (
     extract_create_entities,
```
Comment (shawnsarwar) [inline aether-kernel/aether/kernel/api/views.py:36]: I assume this method is only called if offline extraction is disabled?

**Review by shawnsarwar — DISMISSED** [review]

File: aether-kernel/aether/kernel/api/validators.py (hunk 1 above)
Comment (lordmallam) [reply aether-kernel/aether/kernel/api/validators.py]: Validations are used in entity extraction with the entry point [here](https://github.com/eHealthAfrica/aether-python-library/blob/1.0.3/aether/python/entity/extractor.py#L589). The current validations require the `ValidationError` from `django.core.exceptions` and is the only `django` dependency as used [here](https://github.com/eHealthAfrica/aether-python-library/blob/1.0.3/aether/python/validators.py#L25)

Comment (obdulia-losantos) [reply aether-kernel/aether/kernel/api/validators.py]: For those cases you can create your own `ExtractionError` and catch it in the django module... it's better not to have a django dependency when in fact  you don't need it :thinking:

File: aether-kernel/aether/kernel/api/views.py (hunk 2 above)
Comment (lordmallam) [reply aether-kernel/aether/kernel/api/views.py:36]: Yes that is correct, What I have in this PR is simply the current working kernel implementation using the new python library. No `entity extraction module` link yet. The idea is to break the large `exm` PR  ie: (pythonlib + exm + submissions validation) into smaller PRs. This [PR](https://github.com/eHealthAfrica/aether/pull/649) will be updated to contain the complete abstraction of entity extraction with this as a dependency so as to use the `pythonlib` within `exm`

File: aether-kernel/conf/pip/primary-requirements.txt [hunk 3]
```diff
@@ -28,8 +31,6 @@ python-dateutil
 
 
 # AVRO + JSON libraries
-eha_jsonpath
-jsonschema
 spavro
```
Comment (obdulia-losantos) [inline aether-kernel/conf/pip/primary-requirements.txt]: This is also a requirement in the `aether-python` library ;-)

**Comment by obdulia-losantos** [comment]
:tada: 

```text

Name                                                    Stmts   Miss Branch BrPart  Cover   Missing
---------------------------------------------------------------------------------------------------
aether/__init__.py                                          1      0      0      0   100%
aether/kernel/__init__.py                                   1      0      0      0   100%
aether/kernel/admin.py                                     37      0      0      0   100%
aether/kernel/api/__init__.py                               0      0      0      0   100%
aether/kernel/api/constants.py                              2      0      0      0   100%
aether/kernel/api/entity_extractor.py                      19      0     10      0   100%
aether/kernel/api/exporter.py                             295      0    132      0   100%
aether/kernel/api/filters.py                              119      0     30      0   100%
aether/kernel/api/forms.py                                 15      0      0      0   100%
aether/kernel/api/mapping_validation.py                    71      0     24      0   100%
aether/kernel/api/models.py                               251      0     44      0   100%
aether/kernel/api/project_artefacts.py                    114      0     51      0   100%
aether/kernel/api/serializers.py                          152      0     10      0   100%
aether/kernel/api/urls.py                                  15      0      0      0   100%
aether/kernel/api/utils.py                                 40      0     14      0   100%
aether/kernel/api/validators.py                            17      0      0      0   100%
aether/kernel/api/views.py                                234      0     38      0   100%
aether/kernel/apps.py                                       4      0      0      0   100%
aether/kernel/management/__init__.py                        0      0      0      0   100%
aether/kernel/management/commands/__init__.py               0      0      0      0   100%
aether/kernel/management/commands/extract_entities.py      26      0      6      0   100%
aether/kernel/urls.py                                       5      0      0      0   100%
aether/kernel/views.py                                      9      0      0      0   100%
---------------------------------------------------------------------------------------------------
TOTAL                                                    1427      0    359      0   100%
      ____                 _     _       _     _
     / ___| ___   ___   __| |   (_) ___ | |__ | |
    | |  _ / _ \ / _ \ / _` |   | |/ _ \| '_ \| |
    | |_| | (_) | (_) | (_| |   | | (_) | |_) |_|
     \____|\___/ \___/ \__,_|  _/ |\___/|_.__/(_)

```

**Review by obdulia-losantos — APPROVED** [review]

**Pull request merged**